administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -5,6 +5,8 @@
 
 COM_JOOMLAUPDATE_CAPTIVE_HEADLINE="Confirm your credentials"
 COM_JOOMLAUPDATE_CHECKED_UPDATES="Checked for updates."
+COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_DESC="Shows the checkbox to confirm you have taken a backup and you're ready to update in the final step before the update is actually applied."
+COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_LABEL="Confirm backup checkbox"
```

```suggestion
COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_LABEL="Confirm Backup Checkbox"
```

Follows the style guide